### DISCUSSION TITLE
How to override the footer?

### DISCUSSION THREAD

**Discussion by Andre601** [mmistakes/minimal-mistakes#2990] [Q&A] [answered]
I want to override the footer to include a "Not affiliated" notice to it, making sure that my page is not affiliated with a company it works/relies on.

I so far tried to put `_includes/_footer.html` but had no luck with it. Then I tried `_includes/footer/custom.html` and it works, but A) doesn't override the default footer and B) also looks way too big for the footer.

Just to be clear here will I not remove the notice about the page using jekyll and this theme. All I want is to add that notice I mentioned.

**Comment by mmistakes** [comment, marked as answer]
When you're overriding an include or layout that is bundled with the theme you need to name it exactly the same. Then when you build your site, Jekyll will use your local version in place of what is in the theme.

If you're customizing the footer, copy [`_includes/footer.html`](https://github.com/mmistakes/minimal-mistakes/blob/master/_includes/footer.html) to your repo and make your changes there. Looks like you named it `_footer.html` which is why it didn't do anything for you.

**Reply by Andre601** [reply]
Thanks for the response. I based the name on what was shown in this repository's gh-pages branch (_includes-> _footer.html)

**Reply by mmistakes** [reply]
The gh-pages branch for this repo is super old. It’s from a version of the theme from 2016. 

Everything that applies to the current version is in master. The example site is all built from the `/docs` directory. It doesn’t use the gh-pages branch.
